contrib/platforms/macosx

```diff
@@ -20,15 +20,15 @@ fi
 
 # OS-X 12.2 provides Bison 2.3, we need Bison 3 or better
 # Try to get the 'brew' Bison if installed
-if [ -d /usr/local/opt/bison ]; then
+if [ -d /opt/homebrew/opt/bison ]; then # brew (arm)
+  ENVVARS+=" BISON_ROOT=${BISON_ROOT:-/opt/homebrew/opt/bison}"
+elif [ -d /usr/local/opt/bison ]; then # brew (intel)
   ENVVARS+=" BISON_ROOT=${BISON_ROOT:-/usr/local/opt/bison}"
-fi
-# Try to get the 'MacPort' Bison if installed
-if [ -x /opt/local/bin/bison ]; then
+elif [ -x /opt/local/bin/bison ]; then # macports
   ENVVARS+=" BISON_ROOT=${BISON_ROOT:-/opt/local}"
-fi
-# Try to get the 'Fink' Bison if installed
```

Comment:
Can we keep this comment? I think it would be helpful for future us to know what these directories refer to (also for the two brew lines above)

Reply:
each line us already commented